modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -150,3 +151,28 @@ spec:
 == Fluentd log handling when the external log aggregator is unavailable
 
 If your external logging aggregator becomes unavailable and cannot receive logs, Fluentd continues to collect logs and stores them in a buffer. When the log aggregator becomes available, log forwarding resumes, including the buffered logs. If the buffer fills completely, Fluentd stops collecting logs. {product-title} rotates the logs and deletes them. You cannot adjust the buffer size or add a persistent volume claim (PVC) to the Fluentd daemon set or pods.
+
+[discrete]
+== Supported Authorization Keys
+Common key types are provided here. Some output types support additional specialized keys, documented with the output-specific configuration field. All secret keys are optional. Enable the security features you want by setting the relevant keys. You are responsible for creating and maintaining any additional configurations that external destinations might require, such as keys and secrets, service accounts, port openings, or global proxy configuration. Open Shift Logging will not attempt to verify a mismatch between authorization combinations.
+
+Simple Authentication Security Layer (SASL)::
```

Comment:
Minor: suggest putting SASL last, it's currently only supported by the kafka output (though may be supported by others in future.) TLS and user/pass are relevant to a wider audience.